### ISSUE TITLE
Issue with the Warnings

### ISSUE THREAD

**Issue by Chokerino** [MDAnalysis/membrane-curvature#87]
Hello,

For a particular system, I am getting the following error

```
MDAnalysis.MDAKit.membrane_curvature: WARNING  Atom coordinates exceed size of grid and element (25,13) can't be assigned. Maximum (x,y) coordinates must be < (137.42520141601562, 137.42520141601562). Skipping atom.
```

What does this exactly mean? I am not able to understand why this warning is raised because the atoms are selected from the PDB files specified so no atom should really be outside of the grid.

**Comment by ojeda-e** [comment]
Hi @Chokerino, my apologies for the late reply. I didn't see your message earlier.

The message is indicating that there is an atom of coordinates `x, y = (137.42520141601562, 137.42520141601562)` that exceeds the grid boundaries created by MembraneCurvature and therefore that atom is not taken into account to calculate the surfaces. The message is intended to warn the user about a potential loss in sampling; the more atoms in the `AtomGroup` of reference, the more accurate the derived surface will be. In your case, if your AtomGroup has N atoms, then the warning means that the surfaces are derived from N-1 instead because the atom of the coordinates specified in the message is skipped. 

With that said, from the message I infer that:
- There is an atom of coordinates (x,y) ~ (137.425, 137.425) in your atom group.
- The grid has 25 bins in x and 13 bins in y.

It's possible that there is an atom in the system that exceeds the boundaries of the grid: 137.425 (atom) > 13.0 (grid), and the surface is not using that atom for sampling. If the number of atoms skipped << number of atoms in `AtomGroup` it's ok.  Otherwise, please consider that low sampling may result in inaccurate surfaces and curvatures.

**Comment by ojeda-e** [comment]
Since there is no more activity, I am closing this issue.
